pkg/operator/ceph/object/bucket/controller.go

```diff
@@ -53,6 +55,10 @@ type ReconcileBucket struct {
 // Add creates a new Ceph CSI Controller and adds it to the Manager. The Manager will set fields on the Controller
 // and Start it when the Manager is Started.
 func Add(mgr manager.Manager, context *clusterd.Context, opManagerContext context.Context, opConfig opcontroller.OperatorConfig) error {
+	if os.Getenv(object.RequireObjectCRDs) == "false" {
+		logger.Info("skip running Object Bucket controller")
+		return nil
+	}
```

Comment:
Why not make this into an operator config and use `opConfig.Parameters` here? That is the intended way of passing configs to controllers. @travisn thoughts?

See https://github.com/rook/rook/blob/7f0c83ad18346e8880a168468899be0ec55bc18a/pkg/operator/ceph/csi/controller.go#L176-L179